### PR TITLE
wxwidgets & filezilla rebuild & omnibus update

### PR DESCRIPTION
**Pull request by satmandu**
- Switched `wxwidgets` 3.2.x back to autotools, since the cmake build misconfigures the lib directory on x86_64.
- Updates `libfilezilla`, rebuilds `filezilla` pointing it to `wxwidgets` 3.2.x
- Updates `dav1d`, and consumers of `dav1d`: `libavif`, `libheif`, `ffmpeg`, `webkit2gtk_4_1`
- Updates `libva`
- Security update for `libcue`
- Removed `gtk2` dependencies for `gtk3` apps.
- Switched major common Python dependencies for GUI apps to build dependencies only.
- armv7l `webkit2gtk_4` & `webkitgtk_6` builds in progress. (Those will be in a future PR.)
- `pipewire` updated to 0.3.81

Works properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=wxwidgets_update CREW_TESTING=1 crew update
```
image

### CODE REVIEW

File: packages/filezilla.rb [hunk 1]
```diff
@@ -3,36 +3,46 @@
...
+  depends_on 'libidn2' => :build
+  depends_on 'libwebp' => :build
+  depends_on 'mesa' => :build
+  depends_on 'nettle' # R
+  depends_on 'pango' # R
+  depends_on 'sqlite' => :build
+  depends_on 'sqlite' # R
+  depends_on 'wayland_protocols' => :build
+  depends_on 'wxwidgets' => :build
+  depends_on 'wxwidgets' # R
```
Comment (uberhacker) [inline packages/filezilla.rb:42]: Duplicate depends_on lines?

File: packages/filezilla.rb [hunk 2]
```diff
@@ -3,36 +3,46 @@
...
-  depends_on 'xdg_utils'
-  depends_on 'libwebp'
-  depends_on 'wayland_protocols'
-  depends_on 'mesa'
-  depends_on 'xcb_util'
+  depends_on 'at_spi2_core' # R
+  depends_on 'boost' => :build
+  depends_on 'cairo' # R
+  depends_on 'dbus' => :build
+  depends_on 'dbus' # R
```
Comment (uberhacker) [inline packages/filezilla.rb:23]: Duplicate depends_on lines?

File: packages/filezilla.rb [hunk 3]
```diff
@@ -3,36 +3,46 @@
 class Filezilla < Package
   description 'FileZilla Client is a free FTP solution.'
   homepage 'https://filezilla-project.org/'
-  version '3.65.0'
+  version '3.65.0-1'
   license 'GPL-2'
-  compatibility 'aarch64,armv7l,x86_64'
+  compatibility 'x86_64'
```
Comment (uberhacker) [inline packages/filezilla.rb:8]: What happened to arm?

File: packages/webkit2gtk_4_1.rb [hunk 4]
```diff
@@ -3,21 +3,17 @@
 class Webkit2gtk_4_1 < Package
   description 'Web content engine for GTK'
   homepage 'https://webkitgtk.org'
-  version '2.40.5'
+  version '2.42.1'
   license 'LGPL-2+ and BSD-2'
-  compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://webkitgtk.org/releases/webkitgtk-2.40.5.tar.xz'
-  source_sha256 '7de051a263668621d91a61a5eb1c3771d1a7cec900043d4afef06c326c16037f'
+  compatibility 'x86_64'
```
Comment (uberhacker) [inline packages/webkit2gtk_4_1.rb:8]: Is arm dead?

File: packages/wxwidgets.rb [hunk 5]
```diff
@@ -34,49 +32,70 @@ class Wxwidgets < Package
...
   depends_on 'zlibpkg' # R
 
-  no_env_options
-
   def self.preflight
     %w[wxwidgets30 wxwidgets31].each do |wxw|
       next unless File.exist? "#{CREW_PREFIX}/etc/crew/meta/#{wxw}.filelist"
 
-      puts "#{wxw} installed and conflicts with this version.".orange
+      puts "#{wxw} installed and conficts with this version.".orange
```
Comment (uberhacker) [inline packages/wxwidgets.rb:65]: Original spelling correct?

File: packages/webkit2gtk_4_1.rb (hunk 4 above)
Comment (satmandu) [reply packages/webkit2gtk_4_1.rb:8]: Still building

File: packages/wxwidgets.rb (hunk 5 above)
Comment (satmandu) [reply packages/wxwidgets.rb:65]: yes!

File: packages/filezilla.rb (hunk 1 above)
Comment (satmandu) [reply packages/filezilla.rb:40]: fixed.

File: packages/filezilla.rb (hunk 2 above)
Comment (satmandu) [reply packages/filezilla.rb:22]: Fixed.

File: packages/webkit2gtk_4_1.rb (hunk 4 above)
Comment (satmandu) [reply packages/webkit2gtk_4_1.rb]: Built...

File: packages/filezilla.rb (hunk 3 above)
Comment (satmandu) [reply packages/filezilla.rb]: Built!

**Review by uberhacker — APPROVED** [review]

**Pull request merged**